app/components/post_draft/post_draft.tsx

```diff
@@ -74,7 +74,7 @@ function PostDraft({
         ios: (keyboardHeight ? keyboardHeight - keyboardAdjustment : (postInputTop + insetsAdjustment)),
         default: postInputTop + insetsAdjustment,
     });
-    const autocompleteAvailableSpace = containerHeight - autocompletePosition - (isChannelScreen ? headerHeight + insets.top : 0);
+    const autocompleteAvailableSpace = containerHeight - autocompletePosition - (isChannelScreen ? headerHeight : 0);
```

Comment:
why this change?

Reply:
@elias, for all of the components that have `insets.top` removed, I did a refactor.  

Each component that used `useDefaultHeader` was having to account for `insets.top` inside the component.  I added the `insets.top` to the return value from `useDefaultHeader` so that each component doesn't need to account for that value.

Reply:
This is from refactor in eaad1c3

Reply:
lol my gh username is @enahum ;)

Reply:
`elias username` sorry for tagging you in this PR if you see this :) 
  
  @enahum 🤣.